server/modelpath.go

```diff
@@ -64,7 +64,11 @@ func ParseModelPath(name string) ModelPath {
 		mp.Tag = tag
 	}
 
-	return mp
+	if mp.Repository == "" {
```

Comment:
Which case would trigger this? something like `ollama.ai/:q4_0`?

Reply:
Removed this since it's not actually needed